README.md

```diff
@@ -139,12 +135,14 @@ jobs:
             -d 'grant_type=client_credentials' \
             -d 'scope=2ff814a6-3304-4ab8-85cb-cd0e6f879c1d%2F.default' \
             -d 'client_secret=${{ secrets.AZURE_SP_CLIENT_SECRET }}' |  jq -r  '.access_token')" >> $GITHUB_ENV
-      - name: Trigger notebook from PR branch
+      - name: Trigger model training notebook from PR branch
         uses: databricks/run-notebook@v0
         with:
-          local-notebook-path: notebooks/MainNotebook.py
-          # Alternatively, specify an existing-cluster-id to run against an existing cluster.
-          # The cluter JSON below is for Azure Databricks. On AWS and GCP, set
+          # Run our notebook against a remote repo
```

Comment:
```suggestion
```
This line (I know it was there before) seems a little confusing / not necessary (what does it mean to "run the notebook against a remote repo"?)